SchedulerJobs/SchedulerJobs.Services/CloseConferenceService.cs

```diff
@@ -31,6 +34,29 @@ public async Task<int> CloseConferencesAsync()
 
             return conferenceCount;
         }
+
+        public async Task<int> DeleteAudiorecordingApplicationsAsync()
+        {
+            var conferences = await _videoApiService.GetExpiredAudiorecordingConferences();
+            var conferencesCount = 0;
```

Comment:
This is not thread safe, I know you will say it will be kicked off only once, but nothing is guaranteed in life

Reply:
object not being thread-safe is when a single instance (or static) is accessed by multiple threads at the same time. Do we?

Reply:
Maybe ok here for a Azure function, not 100% sure